services/battle-api/package.json

```
@@ -12,58 +12,26 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-<<<<<<< HEAD
-    @apollo/server: "4.11.3",
-    @graphql-tools/schema: "10.0.30",
-=======
-<<<<<<< HEAD
     "@apollo/server": "4.11.3",
```

Biome (2.4.7)

[error] The key `@apollo/server` was already declared.

(lint/suspicious/noDuplicateObjectKeys)